bin/main.rb

```diff
@@ -1,3 +1,32 @@
 #!/usr/bin/env ruby
 
-puts 'Hello world!'
+puts 'Wellcome to Roby Tic Tac Toe!'
+
+puts 'Enter the First Player Name:'
+first_player_name = gets.chomp
+puts 'Enter the Second Player Name:'
+second_player_name = gets.chomp
+
+puts 'Lets start'
+
+puts '+---+---+---+'
+puts '| 1 | 2 | 3 |'
+puts '+---+---+---+'
+puts '| 4 | 5 | 6 |'
+puts '+---+---+---+'
+puts '| 7 | 8 | 9 |'
+puts '+---+---+---+'
+
+puts "First Player is #{first_player_name} and starts with X"
+
+puts 'Please select a avaiable cell from the board'
+move = gets.chomp.to_i
+
+while move.zero?
+  puts 'Enter a valid value'
+  move = gets.chomp.to_i
+end
+
+system('clear')
+
+puts "Win Player 2 is #{second_player_name}"
```

Comment:
[MANDATORY] 
- This output message comes after the first move no matter what move the first player makes. This is because you don't have a while loop that alternates the turns between the two players while game termination conditions are not met. 
- You can have pseudo 'win' and 'draw' as termination conditions.
  eg: - you can make picking some number like '3' is the winning condition and the draw condition is when the number of turns equals nine, for now. You don't need actual logic here. 

- For visual aid look at the .gif file below and learn how your game should behave on this milestone 

![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone2/game_flow_example.gif)